CauldronMods/Controller/Environments/FSCContinuanceWanderer/Cards/TemporalReversalCardController.cs

```diff
@@ -64,5 +68,32 @@ public override void AddTriggers()
             base.AddEndOfTurnTrigger((TurnTaker turnTaker) => turnTaker == base.TurnTaker, base.DestroyThisCardResponse, TriggerType.DestroySelf);
         }
 
+        public override CustomDecisionText GetCustomDecisionText(IDecision decision)
+        {
+
+            return new CustomDecisionText($"Select the {GetAdjective(deckAdjectiveCount)} deck to return a card to the top of.", $"Select the { GetAdjective(deckAdjectiveCount) } deck to return a card to the top of.", $"Vote for the {GetAdjective(deckAdjectiveCount)} deck to return a card to the top of.", "return a card to deck");
+
+        }
+
+        private string GetAdjective(int deckAdjectiveCount)
```

Comment:
We have an extension method for this: int.ToOrdinalString()

Reply:
Ah I was looking through the extension methods and I didn't see that - thank you! I'll adjust it.